README.md

```diff
@@ -9,40 +9,106 @@ and name or excluded keys.
 
 ## Configuration
 
-By default, plugin will localize source `.yml` files,
+By default, plugin will localize source `.yml` and `.yaml` files,
 e.g. `/project/en.yml`,  and write localized file
-to a separate folder: `/project/<localeName>/en.yml`.
+to a subfolder: `/project/resources/ru-RU/en.yml`.
+
+The plugin will look for the `yaml` property within the `settings`
+of your `project.json` file. The following settings are
+used within the json property:
+
+- mappings: a mapping between file matchers and an object that gives
+  info used to localize the files that match it. This allows different
+  json files within the project to be processed with different schema.
+  The matchers are
+  a [micromatch-style string](https://www.npmjs.com/package/micromatch),
+  similar to the the `includes` and `excludes` section of a
+  `project.json` file. The value of that mapping is an object that
+  can contain the following properties:
+  - schema: schema file to use with that matcher. See more information
+    about schema files below.
```

Comment:
I would prefer that when you use a mapping, then you don't need a schema file at all. That is, all of the properties that appear in the schema file would appear here in the mappings object instead. But for backwards compatibility, when you don't use a mapping and there is an old schema file, it uses the schema file in the old way. That is, if you don't update your project.json file, then the plugin operates the same way it always has.